main.go

```diff
@@ -1,21 +1,73 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
 	"os"
+	"os/user"
+	"path/filepath"
 
 	"github.com/abs-lang/abs/repl"
+	"github.com/hashicorp/go-getter"
 )
 
 var Version = "1.8.0"
 
+func check(e error) {
+	if e != nil {
+		panic(e)
```

Comment:
I'd like to print a beautified error message rather than a raw panic. After we print, we can os.Exit with an error code to stop the program